### PR TITLE
10: Allow nwn client to stop on signal and update to newest SDK to al…

### PR DESCRIPTION
**Pull request by TNO-SlaFleur**
…low RabbitMQ to retry connections.

### CODE REVIEW

File: src/optimizer_worker/main.py
Comment (MarkTNO) [inline src/optimizer_worker/main.py:1]: Better to only use nwn_client main functions? Not use deeper function in db_client or broker_client, less straightforward.
So functions called on line 78, 79 and 84 should then be added in nwn_client.py in sdk.

File: test.py [hunk 1]
```diff
@@ -28,26 +28,29 @@
...
+        user_name="fleursl",
+        project_name="some-project",
+    )
+
+    print(f"Polling status of job {job_id} until finished.")
+
+    unfinished = True
+    while unfinished:
+        time.sleep(5)
+        job = nwn_client.db_client.get_job(job_id)
```
Comment (MarkTNO) [inline test.py:46]: change function call to get_job_details on nwn_client

**Comment by TNO-SlaFleur** [comment]
Reminder: confirm if reconnects are working as well.

File: test.py (hunk 1 above)
Comment (TNO-SlaFleur) [reply test.py]: Fixed!

File: src/optimizer_worker/main.py
Comment (TNO-SlaFleur) [reply src/optimizer_worker/main.py:1]: Fixed!

**Pull request merged**